### ISSUE TITLE
Support forward proxy

### ISSUE THREAD

**Issue by romanholidaypancakes** [vpnhood/VpnHood#118]
In my company I need a proxy to access the network, so support http/socket fronting for vpnhood to work

**Comment by trudyhood** [comment]
Hi, Thank you for your interest in VpnHood.
Would you describe more about your requirements? I couldn't understand it.

**Comment by vochong** [comment]
> as described in this article [Connecting to an OpenVPN server via an HTTP proxy.](https://openvpn.net/community-resources/connecting-to-an-openvpn-server-via-an-http-proxy/) `socks/http(s) -> vpnhood(openvpn) -> internet`

I fully agree that VpnHood should definitely support HTTP proxy.  It will make it work in virtually any network environment and become a very versatile VPN solution.   Thanks a lot trudyhood for all your effort.
